README.md

```diff
@@ -31,6 +31,9 @@ Tokens should be able to contain any assertions (claims) that a developer wants
 
 Guardian also allows you to configure multiple token types/configurations in a single application.
 
+## Documentation
+API documentation is available at [https://hexdocs.pm/guardian](https://hexdocs.pm/guardian)
```

Comment:
Probably is better to add a jump line here so we keep consistency.

Reply:
done